<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageFrame</class>
 <widget class="QFrame" name="MessageFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: transparent;</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>75</y>
     <width>750</width>
     <height>450</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="BUTTON_OK">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>370</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BUTTON_CANCEL">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>370</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="LABEL_MESSAGE">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>80</y>
      <width>740</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="LABEL_TITLE">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>10</y>
      <width>740</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
